src/main/java/com/ericsson/ei/controller/RuleCheckControllerImpl.java

```diff
@@ -120,21 +121,22 @@
                 if (aggregatedObject != null && !aggregatedObject.equals("[]")) {
                     return new ResponseEntity<>(aggregatedObject, HttpStatus.OK);
                 } else {
-                    String errorMessage = "Aggregated event is not generated. List of rules or list of events are not correct";
+                    String message = "Aggregated event is not generated. List of rules or list of events are not correct";
+                    String errorMessage = "Failed to generate aggregated object. Error message:\n" + message;
```

Comment:
I don't think you need to write "Error message:" here since it's something we write ourselves. I believe we have written that when we have used e.getMessage.

Reply:
Done.